09. cli-and-http-request/clihelper/PersonTypeStruct.go

```diff
@@ -0,0 +1,13 @@
+package clihelper
+
+import "time"
+
+type PersonType struct {
```

Comment:
We will move this `person` this person struct to the outside of this package.
As it is not part CLI but only an implementation of the outside

Reply:
okay